doc/guides/debug-symbols.md

```diff
@@ -0,0 +1,29 @@
+# Debug Symbols
+
+Debug symbols are constants present in the final build which can be used by
+debuggers or other scripts to navigate internal structures of the software when
+analyzing its memory. This is useful when doing postmortem analysis, for
+example. Node provides debug symbols in its builds for V8 and Node internal
+structures.
+
+## V8 Debug Symbols
+
+All symbols available for V8 are prefixed with `v8dbg_`, and they allow
+inspection of objects in the heap as well as those object's properties and
+references. Those symbols are provided by V8 itself by the script
+`deps/v8/tools/gen-postmortem-metadata.py`.
```

Comment:
If this is intended for maintainers and we do want to refer to the script, you might want to mention that the script contains additional information about the various metadata that exists.

Reply:
If it's in `doc/guides`, then it should be for maintainers, user guides go in [nodejs/nodejs.org](https://github.com/nodejs/nodejs.org/).

Reply:
This document is intended for maintainers, but we could add a guide to nodejs/nodejs.org targeting users and tools developers. I'll update this document to make it clearer and more helpful.